test/rosenbrock.jl

```diff
@@ -164,3 +173,188 @@ _p = [1.0, 100.0]
 f = OptimizationFunction(rosenbrock,ModelingToolkit.AutoModelingToolkit(),x0,_p,grad=true,hess=true)
 prob = OptimizationProblem(f,x0,_p)
 sol = solve(prob,Optim.Newton())
+
+
+### Nonconvex test
+prob = GalacticOptim.OptimizationProblem(f, x0, _p, lb = [-1.0,-1.0], ub = [1.5,1.5])
+
+sol = solve(prob, MMA02())
+@test 10*sol.minimum < l1
+
+sol = solve(prob, GCMMA())
```

Comment:
MMA02 and GCMMA are aliases. You probably wanted to test `MMA87()`.

Reply:
I had both of them to make sure it handles both notation. `MMA87()` seems to struggle with the rosenbrock function nay idea why?